repos/system_upgrade/common/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -242,3 +260,47 @@ def _get_firewall_status(service_name):
 def get_firmware():
     firmware = 'efi' if os.path.isdir('/sys/firmware/efi') else 'bios'
     return FirmwareFacts(firmware=firmware)
+
+
+@aslist
+def _default_grub_info():
+    default_grb_fpath = '/etc/default/grub'
+    if not os.path.isfile(default_grb_fpath):
```

Comment:
@Rezney aaah...I think I wanted to ask @smitterl  abou existence of this file on s390x... as this one is created during installation of the system. I expect the file does not exist on s390x. but better to check.

Reply:
It does not exist.